File: Techievibe.DataAccess/Models/BlogPosts.cs
using System;
using System.Collections.Generic;

namespace Techievibe.DataAccess.Models
{
    public partial class BlogPosts
    {
        public int PostId { get; set; }
        public string PostTitle { get; set; }
        public string PostBody { get; set; }
        public string PostAuthor { get; set; }
        public DateTime PostDate { get; set; }
        public int PostReadingTime { get; set; }
        public int? PostLikeCount { get; set; }
        public int? PostCommentCount { get; set; }
        public int PostCategoryId { get; set; }

        public BlogPostCategories PostCategory { get; set; }
    }
}

File: Techievibe.Ui/src/app/models/blog-posts.ts
import { PostCategories } from './post-categories';

export class BlogPosts {
    constructor(
        public postId: number,
        public postTitle: string,
        public postBody: string,
        public postAuthor: string,
        public postDate: string,
        public postReadingTime: number,
        public postLikeCount: number, 
        public postCommentCount: number,
        public postCategoryId: number,
        public postCategory: PostCategories
        ){}
}